Project/src/main/resources/static/js/upload.js

```diff
@@ -4,17 +4,61 @@ function checkFileSize(event) {
     var fileArr = fileElement.files;
 
     if (!fileArr || !fileArr[0]) {
-        alert("No file loaded");
+        alertUpload("ERROR: No File Selected")
         event.preventDefault();
         return;
     }
 
     var file = fileArr[0];
 
     if (file.size > fileSizeLimit) {
-        alert("File too large");
+        alertUpload("ERROR: File Too Large");
+        event.preventDefault();
+        document.getElementById("fileUploadForm").reset();
+        return;
+    } else if (file.size <= 0) {
+        alertUpload("ERROR: File Contains no Data");
         event.preventDefault();
         document.getElementById("fileUploadForm").reset();
         return;
     }
-}
\ No newline at end of file
+
+}
+
+function alertUpload(errorMsg) {
+    document.getElementById("errorMsg").innerText = errorMsg;
+    $('.alert').show();
+}
+
+
+$(function() {
+
+    // We can attach the `fileselect` event to all file inputs on the page
+    $(document).on('change', ':file', function() {
+        var input = $(this),
+            numFiles = input.get(0).files ? input.get(0).files.length : 1,
+            label = input.val().replace(/\\/g, '/').replace(/.*\//, '');
+        input.trigger('fileselect', [numFiles, label]);
+    });
+
+    // We can watch for our custom `fileselect` event like this
+    $(document).ready( function() {
```

Comment:
isnt this redundant as  $(function() {}) on line 34 does the same thing?

Reply:
tbh I just copied the code and didn't even read it but yeah you right lol.